master/security/namespace-policy.md

```diff
@@ -60,7 +63,31 @@ spec:
       ports:
       - 6379
 ```
+
+#### Use Kubernetes RBAC to control namespace label assignment
+
```

Comment:
Network policies can be applied to endpoints using selectors that match labels on either the endpoint itself, the endpoint's namespace, or the endpoint's service account. If you use selectors based on the endpoint's namespace, you can use Kubernetes RBAC to control which users can assign labels to namespaces. This allows you to separate groups who are allowed to deploy pods from those who can assign labels to namespaces.

In the following example, users in the **development** environment are allowed to communicate only with other pods that have namespace labels `environment == "development"`.

(FYI...I updated my build and this policy is still failing for me...)

Reply:
Fixed